packages/client/src/Config.ts

```diff
@@ -26,9 +25,113 @@ export interface PrivateKeyAuthConfig {
     address?: string
 }
 
-export interface TrackerRegistryContract {
-    jsonRpcProvider?: ConnectionInfo
-    contractAddress: string
+export interface Layer0Config {
+
+    /**
+     * The list of entry point PeerDescriptors used to join the Streamr Network.
+     */
+    entryPoints?: JsonPeerDescriptor[]
+
+    /**
+     * The list of STUN and TURN servers to use in ICE protocol when
+     * forming WebRTC connections.
+    */
+    iceServers?: IceServer[]
+
+    /**
+     * When set to true private addresses will not be probed when forming
+     * WebRTC connections.
+     *
+     * Probing private addresses can trigger false-positive incidents in
+     * some port scanning detection systems employed by web hosting
+     * providers. Disallowing private addresses may prevent direct
+     * connections from being formed between nodes using IPv4 addresses
+     * on a local network.
+     *
+     * Details: https://github.com/streamr-dev/network/wiki/WebRTC-private-addresses
+    */
+    webrtcDisallowPrivateAddresses?: boolean
+
+    /**
+     * Defines WebRTC connection establishment timeout in milliseconds.
+     *
+     * When attempting to form a new connection, if not established within
+     * this timeout, the attempt is considered as failed and further
+     * waiting for it will cease.
+    */
+    newWebrtcConnectionTimeout?: number
```

Comment:
Could use the same naming convention as in other WebRTC-related field: i.e. rename to `webrtcNewConnectionTimeout`